src/opm/output/eclipse/AggregateGroupData.cpp

```diff
@@ -847,8 +847,14 @@ void staticContrib(const Opm::Group&        group,
 
     if (glo.has_group(group.name())) {
         const auto& glo_group = glo.group(group.name());
-        sGrp[Isp::GLOMaxSupply] = sgprop(M::gas_surface_rate, glo_group.max_lift_gas().value());
-        sGrp[Isp::GLOMaxRate]   = sgprop(M::gas_surface_rate, glo_group.max_total_gas().value());
+
+        if (const auto& max_supply = glo_group.max_lift_gas(); max_supply.has_value()) {
```

Comment:
There is a default value here, so I would suggest the `else` clause:
```C++
...
else
     sGrp[Isp::GLOMaxRate] = ParserKeywords::GLIFTOPT::MAX_TOTAL_GAS_RATE::defaultValue;
```

Reply:
> There is a default value here, so I would suggest the else clause:

It Depends[TM], and I really don't know the correct answer at the moment.  By default we do (and should) output `0.0` unless there's gas-lift optimisation.  I'll need to defer to someone else to ascertain the correct output value for the case of "gas lift optimisation active, but no limit applies to a given group".

Reply:
Have found a "someone" to do the check?

Reply:
>> I'll need to defer to someone else to ascertain the correct output value for the case of "gas lift optimisation active, but no limit applies to a given group".
>
> Have you found a "someone" to do the check?

I have not.